tests/test_config.py

```diff
@@ -925,3 +791,37 @@ def test_apply_config(self):
         self.assertEqual(
             config.apply_config(api=results_api, config=configuration), results_api
         )
+
+
+class TestEasySASTConfig(CLITestCase):
+    """
+    Test config.py
+    """
+
+    ## create_arg_parser tests
+    # pylint: disable=too-many-statements
+    def test_create_arg_parser(self):
+        """
+        Test the create_arg_parser function
+        """
+
+        # Succeed when calling the create_arg_parser function and do not pass
+        # --verbose or --debug as arguments
+        output = self.parser.parse_args([])
+        self.assertIsNone(output.loglevel)
+
+        # Succeed when calling the create_arg_parser function and pass only
+        # --verbose as an argument
+        output = self.parser.parse_args(["--verbose"])
+        self.assertEqual(output.loglevel, "INFO")
+
+        # Succeed when calling the create_arg_parser function and pass only
+        # --debug as an argument
+        output = self.parser.parse_args(["--debug"])
+        self.assertEqual(output.loglevel, "DEBUG")
+
+        # Fail when calling the create_arg_parser function and pass both
+        # --debug and --verbose as an argument, as they are mutually exclusive
+        with self.assertRaises(SystemExit) as contextmanager:
+            output = self.parser.parse_args(["--debug", "--verbose"])
+        self.assertEqual(contextmanager.exception.code, 2)
```

Comment:
Following your pattern to split items between easy_sast global and veracode specific, we should add the global `--version` tests here.  In the future we may split versions between the veracode plugin and easy_sast, or maybe we won't.  Open to feedback on approach - perhaps in this PR we remove from the veracode tests and add back in if there's a need?

```suggestion
        self.assertEqual(contextmanager.exception.code, 2)

        # Succeed when calling the create_arg_parser function and pass only
        # --version as an argument
        with self.assertRaises(SystemExit) as contextmanager:
            output = self.parser.parse_args(["--version"])
        self.assertEqual(contextmanager.exception.code, 0)
        # Using sys instead of _sys because it is the same thing per
        # https://github.com/python/cpython/blob/6a263cf1adfc18cdba65c788dd76d35997a89acf/Lib/argparse.py#L90
        mock__print_message.assert_called_once_with(veracode_version + "\n", sys.stdout)

        # Succeed when calling the create_arg_parser function and do not pass
        # --version as an argument
        output = self.parser.parse_args(["--verbose"])
        with self.assertRaises(SystemExit) as contextmanager:
            output = self.parser.parse_args(["--version"])
        self.assertEqual(contextmanager.exception.code, 0)
```

Reply:
I'm good to split this off into the global EasySAST config and re-add plugin specific version tests later if needed.